modules/telco-core-cpu-partitioning-and-performance-tuning.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-cpu-partitioning-and-performance-tuning_{context}"]
+= CPU partitioning and performance tuning
+
+New in this release::
+* No reference design updates in this release
+
+Description::
+CPU partitioning separates sensitive workloads from general-purpose tasks, interrupts, and driver work queues, improving performance and reducing latency.
+The CPUs allocated to those auxiliary processes are referred to as _reserved_ in the following sections.
+In Hyper-Threaded systems, a CPU is one hyper-thread.
```

Comment:
Does "Hyper-Threaded" really need to be capitalized? It's not in some other places (though it is in others). I'd double check all throughout

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#hyper-threading

I think "hyper-threaded" is better here. "Hyper-Threading" when referring to the Intel technology specifically.